app-misc/mdate/mdate-1.7.0.3.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A converter to Mayan dates"
+DESCRIPTION="Mdate is a utility that converts Gregorian dates to Mayan Long Count dates. \
+It was originally an implementation of some algorithms published by N. Dershowitz and \
+E.M. Reingold in their book Calendrical Calculations."
+HOMEPAGE="https://github.com/ewe2/mdate"
+COPYRIGHT="2010 - 2014 Sean Dwyer"
```

Comment:
We prefer to have
```
COPYRIGHT="2010-2014 Sean Dwyer"
```
Would you like me to push this little change for you?
Or would you prefer to do it yourself?
Thanks!